compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/ComposeBridge.desktop.kt

```diff
@@ -301,9 +303,15 @@ internal abstract class ComposeBridge {
     }
 
     protected fun updateSceneSize() {
+        val scale = scene.density.density
+        val size = IntSize(
+            width = (component.width * scale).roundToInt(),
+            height = (component.height * scale).roundToInt()
```

Comment:
Why the change from `toInt` to `roundToInt`?

Reply:
`roundToInt` in iOS/macOS is to prevent float rounding errors, because the size is float there.

`toInt` here is to not cut the content in case if we have non-integer density:
210*1.25 = 262.75

We shouldn't adapt the formula to other platforms, we should adapt it to how Swing internally converts the size of the window in pixels to int points.

It is impossible to do precisely, so we have 2 evils - have 1px blank line (`toInt`), or sometime cut the content by 1px (`roundToInt`, sometimes it cuts, sometimes it adds a blank line). `toInt` is more stable here - it just always adds a blank line

Reply:
Reverted